### ISSUE TITLE
Documentation Insufficient

### ISSUE THREAD

**Issue by ghost** [richsage/RMSPushNotificationsBundle#52]
We should add better documentation for this bundle. iOS Feedback is not specified in README docs, or how to handle responses, from both APN and GCM servers.

**Comment by pculka** [comment]
Is it possible to make the documentation usable for GCM?

**Comment by richsage** [comment]
@pculka what specifically did you need adding? The concept is:

    $message = new AndroidMessage()
    $message->setGCM(true);
    $message->setDeviceIdentifier("yourGCMtoken");
    $message->setData(["foo" => "bar"]);

    $container->get('rms_push_notifications')->send($message);

**Comment by pculka** [comment]
@richsage  yup, that exactly :) thanks :) was already browsing through the code how to properly set the data etc what to send. thanks

**Comment by richsage** [comment]
@pculka ah great - that is all in the README, but split into an iOS example and the `setGCM()` bit separately :-)

**Comment by pculka** [comment]
@richsage well in the iOS you are using  `$message->setMessage('Oh my! A push notification!');` instead `$message->setData([]);` and the difference in those is confusing

**Comment by richsage** [comment]
@pculka ah, with you now! Sure - I'll make a note to put that above snippet in the README then in that case.
